src/open_inwoner/openzaak/tests/test_import_export.py

```diff
@@ -263,8 +263,12 @@ def test_export_catalogus_configs_as_jsonl(self):
 class TestCatalogusImport(TestCase):
     def setUp(self):
         self.storage = InMemoryStorage()
-        self.service = ServiceFactory(slug="service-0")
-        ServiceFactory(slug="service-1")
+        self.service = ServiceFactory(
+            slug="service-0", api_root="https://foo.0.maykinmedia.nl"
```

Comment:
```suggestion
            slug="service-foo", api_root="https://foo.maykinmedia.nl"
```
I find `foo` and `bar` more readable than `foo.0` and `foo.1`. Would need to be changed below as well, though.